diagnostic.md

```diff
@@ -6,28 +6,28 @@ Write your answers inside this file, where it's indicated by the comments.
 What command do we use to do that? Use the correct URL for your fork of this
 repository in your answer.
 
-<!-- Remove this comment and place your answer here. -->
+git clone git@github.com:kpie89/git-basics-diagnostic.git
 
 2.  What do you do after cloning a repository, but before starting work?
 
-<!-- Remove this comment and place your answer here. -->
+git init and create a new branch
```

Comment:
You don't want to run `git init` inside cloned a repo - `git init` creates a new repository. Creating a new branch is correct, though.